setup.py

```diff
@@ -81,6 +81,8 @@
     EXTRAS_REQUIRE = {}
     if sys.platform.startswith('win'):
         INSTALL_REQUIRES.append('pyreadline')
+    else:
+        INSTALL_REQUIRES.append('wcwidth')
```

Comment:
Do you only need the **wcwidth** module on non-Windows platforms?

Reply:
Yes